Dis 887 make user agents table toggleable

Jacobomara901 opened the pull request:
https://aspen-discovery.atlassian.net/jira/software/c/projects/DIS/issues/DIS-887

Release notes to follow upon passing QA

AlexanderBlanchardAC committed:
DIS-836: chore: Update change log

AlexanderBlanchardAC committed:
DIS-836: chore: Update change log

AlexanderBlanchardAC committed:
DIS-840: refactor: Update Advertise Page

-Add reward description to advertise page
-Remove the need for rewards to be digital in order for images to
display on advertise page

AlexanderBlanchardAC committed:
DIS-840: refactor: Update Advertise Page

-Add reward description to advertise page
-Remove the need for rewards to be digital in order for images to
display on advertise page

AlexanderBlanchardAC committed:
DIS-840: feat: Add reward descriptions to your campaigns

AlexanderBlanchardAC committed:
DIS-840: feat: Add reward descriptions to your campaigns

AlexanderBlanchardAC committed:
DIS-840: refactor: Update default reward description

AlexanderBlanchardAC committed:
DIS-840: refactor: Update default reward description

AlexanderBlanchardAC committed:
DIS-840: chore: Update change log

AlexanderBlanchardAC committed:
DIS-851: feat: Remove enrolled campaigns from other tables

- When a user enrolls in a camapaign remove it from the upcoming and
active campaign tables so that it dispalys only once.

AlexanderBlanchardAC committed:
DIS-851: chore: Update release note

AlexanderBlanchardAC committed:
DIS-838: refactor: Add padding to campaign landing page images

AlexanderBlanchardAC committed:
DIS-838: refactor: Adjust display and add padding in your campaigns

Adjust the display of the reward columns in the your campaigns pages.

AlexanderBlanchardAC committed:
DIS-838: chore: Update change log

NashvillePublicLibrary committed:
DIS-861 Weeding Report template code cleanup

NashvillePublicLibrary committed:
DIS-861 Fix MS Edge Circulation Report problem

NashvillePublicLibrary committed:
DIS-39 Correct session resurrection

LeoStoyanov committed:
DIS-873: Disable the selection of today if all time slots have passed.

NashvillePublicLibrary committed:
DIS-39 Change SnapPay complete flow

Instead of leaving the user at SnapPay/Complete, redirect user to /MyAccount/Fines. This should:
+ fully restore logged in session with appropriate user menus displayed
+ avoid issues in which a users browser reloads /SnapPay/Complete after the payment has been processed

NashvillePublicLibrary committed:
DIS-39 correct request query parameter

NashvillePublicLibrary committed:
DIS-39 correct decryption of passed user_payments.id

NashvillePublicLibrary committed:
DIS-39 Display message from Aspen user_payments.message to user

Abhyastamita committed:
DIS-188: Better handling of cases where series is null or false

Abhyastamita committed:
DIS-188: Don't delete series that still have titles and do remove series from Solr when deleting

Abhyastamita committed:
DIS-188: Update release notes

Jacobomara901 committed:
DIS-887: Add useragent tracking toggle to systemvariables

Jacobomara901 committed:
DIS-887: Add useragent tracking toggle to db

Jacobomara901 committed:
DIS-887: Only track useragents if enabled

Jacobomara901 committed:
DIS-887: Delete old tracking if we toggle useragent tracking off

mdnoble73 committed:
Merge pull request #2513 from K-Alette/25.06.00

DIS-802: Materials Request: Assign and change status at the same time for materials request

mdnoble73 committed:
DIS-802 Add Assigned To as a default field within Materials Requests when updating requests

LeoStoyanov committed:
DIS-873: Omnibus changes to overhaul curbside pickups.

Abhyastamita committed:
DIS-188: set series index settings runFullUpdate to true

This will clean up deleted series records that are still in Solr

AlexanderBlanchardAC commented:
Test plan followed and results show patch is working as expected. Requests to localhost no longer populate these tables once the 'Disable User Agent Tracking' checkbox is ticked.

LeoStoyanov committed:
DIS-493: Add back accidentally deleted code.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-493-prioritize-available-titles-from-koha-25.06.00

LeoStoyanov committed:
DIS-873: Move CurbsidePickups.php back to MyAccount.

mdnoble73 committed:
Merge pull request #2519 from Abhyastamita/DIS-188_Series_Records

DIS-188: Series bug fixes

lathomas64 committed:
DIS-360: looping through copy notes to get title for each note.

LeoStoyanov committed:
DIS-873: Fix class name for proper action routing.

mdnoble73 committed:
Merge branch '25.06.00' into DIS-815-default-sort-manual-grouping-authorities-page-by-date-added-desc

mdnoble73 committed:
Merge pull request #2478 from LeoStoyanovByWater/DIS-815-default-sort-manual-grouping-authorities-page-by-date-added-desc

DIS-815: Default Sort Manual Grouping Authorities & Author Authorities by Date Added Desc

mdnoble73 committed:
Merge pull request #2477 from LeoStoyanovByWater/DIS-803-character-limit-feedback-when-editing-inputs-and-textareas

DIS-803: Add Character Limit Feedback and Validation to Forms

mdnoble73 committed:
DIS-803 Merge Javascript

mdnoble73 committed:
Merge pull request #2479 from thekesolutions/block_actions_for_expired_patrons

DIS-551: Block actions for expired patrons

mdnoble73 committed:
Merge branch 'DIS-813_correct_description_for_user_age' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-813_correct_description_for_user_age

mdnoble73 committed:
Merge branch 'openfifth-DIS-813_correct_description_for_user_age' into 25.06.00

mdnoble73 committed:
Merge branch 'DIS-796_allow_images_for_physical_rewards' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-796_allow_images_for_physical_rewards

mdnoble73 committed:
Merge branch 'openfifth-DIS-796_allow_images_for_physical_rewards' into 25.06.00

mdnoble73 committed:
Merge pull request #2485 from openfifth/DIS-597_sideloads_with_forward_slash_rerouted

DIS-597: Sideloads With Forward Slash Rerouted

mdnoble73 committed:
Merge pull request #2487 from LeoStoyanovByWater/DIS-818-pagination-sorting-improvements-to-usage-pages

DIS-818: Pagination & Sorting Improvements to Usage Pages

mdnoble73 committed:
Merge branch 'DIS-860_update_award_automatically' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-860_update_award_automatically

mdnoble73 committed:
Merge branch 'openfifth-DIS-860_update_award_automatically' into 25.06.00

mdnoble73 committed:
Merge pull request #2500 from lathomas64/DIS-698-Self-Registration-Failures

DIS-698: using getMainLocation() in self registration because branchc…

mdnoble73 committed:
Merge branch '25.06.00' into 25.06.00

mdnoble73 committed:
Merge pull request #2516 from catsoup11789/25.06.00

DIS-800 - Update tabs/spacing

mdnoble73 committed:
Merge pull request #2508 from LeoStoyanovByWater/DIS-866-update-display-name-when-updating-koha-contact-info

DIS-866: Update Display Name When Updating Koha Contact Info

LeoStoyanov committed:
DIS-891: Correct the count of the number of available copies on grouped works.

LeoStoyanov committed:
DIS-891: Release notes.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-891-fix-incorect-copies-available-count-on-grouped-works

myr-onl committed:
DIS-892: Add custom series image to theme settings

myr-onl committed:
Update 25.06.00.MD

LeoStoyanov committed:
DIS-893: Suppress the copies available count if the title has only on-order copies and no holds.

LeoStoyanov committed:
DIS-893: Release notes.

LeoStoyanov committed:
DIS-791: Correct naming of the cron log.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-791-add-threads-to-update-reading-history-java-process

kidclamp committed:
DIS-895: Add series_indexer to IntelliJ project

LeoStoyanov committed:
DIS-864: Check if multiSelect property is a checkBoxList before choosing which branch to process for display.

LeoStoyanov committed:
DIS-864: Release notes.

kidclamp committed:
DIS-895: Release notes

LeoStoyanov committed:
DIS-900: Display subcategories when searching from admin sidebar.

LeoStoyanov committed:
DIS-900: Release notes.

K-Alette committed:
DIS-806: Remove Materials Request updated successfully page

Remove both created/updated successfully pages for materials requests and instead redirect to the appropriate /MyRequests or /ManageRequests page

For most errors on submitting new requests, the user will still be redirected to a new page

Update release notes

K-Alette committed:
Update Submit.php

mdnoble73 committed:
DIS-897 Make it easier to create new Carl.X sites by creating account profiles and indexing profiles from createSite.php

mdnoble73 committed:
Merge pull request #2528 from K-Alette/25.06.00

DIS-806: Remove Materials Request updated successfully page

mdnoble73 committed:
Merge pull request #2522 from myr-onl/25.06.00

DIS-892: Add custom series image to theme settings

mdnoble73 committed:
Merge pull request #2514 from LiYanjun19/DIS-871-Add-Null-Check-For-Hoopla-Scope

DIS-871 Add Null Checks for Hoopla scope

mdnoble73 committed:
Merge pull request #2512 from LeoStoyanovByWater/DIS-870-allows-users-to-upload-gifs

DIS-870: Allows Users to Upload GIFs

mdnoble73 committed:
Merge pull request #2515 from LiYanjun19/DIS-872-Hoopla-Flex-Expiration

DIS-872 Ensure consistent expiration date display for reserved holds

mdnoble73 committed:
DIS-898 Send log level to messages.log during logging

mdnoble73 committed:
Merge branch '25.06.00' of https://github.com/Aspen-Discovery/aspen-discovery into 25.06.00

mdnoble73 committed:
Merge branch 'DIS-862_date_of_birth_in_csv_download' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-862_date_of_birth_in_csv_download

mdnoble73 committed:
Merge branch 'openfifth-DIS-862_date_of_birth_in_csv_download' into 25.06.00

LeoStoyanov committed:
DIS-868: Allow the "X" check digit in ISBN/ISSN/UPC input fields.

mdnoble73 committed:
Merge branch 'DIS-249_allow_use_of_preferred_name' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-249_allow_use_of_preferred_name

mdnoble73 committed:
Merge branch 'openfifth-DIS-249_allow_use_of_preferred_name' into 25.06.00

LeoStoyanov committed:
DIS-868: Only allow digits for the UPC field.

mdnoble73 committed:
Merge pull request #2518 from LeoStoyanovByWater/DIS-876-push-reading-history-entries-with-no-author-to-the-bottom

DIS-876: Reading History “Sort by Author” Should Push Entries with Missing Authors to the Bottom

LeoStoyanov committed:
Merge branch '25.06.00' into restrict-isbn-input-for-materials-requests

mdnoble73 committed:
Merge branch '25.06.00' into DIS-864-special-fields-for-format-not-displaying-corectly

mdnoble73 committed:
Merge pull request #2525 from LeoStoyanovByWater/DIS-864-special-fields-for-format-not-displaying-corectly

DIS-864: Fixed Special Fields for Format Checkboxes Not Displaying Correctly

mdnoble73 committed:
Merge pull request #2524 from bywatersolutions/DIS-895

DIS-895: Add series_indexer to IntelliJ project

LeoStoyanov committed:
DIS-900: Update release notes.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-900-show-subcategories-when-searching-admin-interface

LeoStoyanov committed:
DIS-901: Fix and improve the filtering of user ratings.

Commit:
DIS-825: Set _expired attribute properly

Commit:
DIS-825: Update release notes

LeoStoyanov committed:
DIS-903: Add delay before requesting Symphony to find the newly registered user.

LeoStoyanov committed:
DIS-903: Release notes.

LeoStoyanov committed:
DIS-849: Remove errant curly brace.

AlexanderBlanchardAC committed:
DIS-836: refactor: Add note to setting in library

Add a note to the settings description to warn users that the Web
Builder must be enabled in order for the leaderboard link to show.

AlexanderBlanchardAC committed:
Merge pull request #239 from AlexanderBlanchardAC/DIS-836_admin_control_over_ce_leaderboard

Dis 836 admin control over ce leaderboard

AlexanderBlanchardAC committed:
Merge branch 'DIS-836_admin_control_over_ce_leaderboard' into DIS-836_admin_control_over_ce_leaderboard

AlexanderBlanchardAC committed:
Merge pull request #249 from AlexanderBlanchardAC/DIS-836_admin_control_over_ce_leaderboard

DIS-836: chore: Update change log

AlexanderBlanchardAC committed:
Merge pull request #242 from AlexanderBlanchardAC/DIS-851_do_not_duplicate_campaigns

Dis 851 do not duplicate campaigns

mdnoble73 committed:
DIS-904 - Updates for searching authors with parentheses

AlexanderBlanchardAC committed:
Merge pull request #241 from AlexanderBlanchardAC/DIS_840_display_descriptions

Dis 840 display descriptions

AlexanderBlanchardAC committed:
Merge pull request #250 from AlexanderBlanchardAC/DIS-838_padding_around_reward_image

Dis 838 padding around reward image

LeoStoyanov committed:
DIS-901: Fix browse category sorting and "Add to Browse" sorting.

LeoStoyanov committed:
DIS-901: Release notes.

LeoStoyanov committed:
DIS-901: Rename user rating sorts.

mdnoble73 committed:
DIS-905 Fix list order in LiDA for browse categories based on lists.

LeoStoyanov committed:
DIS-903: Change logs for failed Amazon SES emails to output more information.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-903-welcome-email-not-sent-when-registering-new-user-symphony

Abhyastamita committed:
DIS-720: Include date placed for Sierra holds so that they can be sorted that way

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-873-fix-and-refactor-koha-curbside-pickup

LeoStoyanov committed:
DIS-873: Release notes.

mdnoble73 committed:
Merge pull request #2532 from LeoStoyanovByWater/DIS-849-web-builder-static-location-and-hours

DIS-849: Remove Errant Curly Brace

mdnoble73 committed:
Merge pull request #2535 from openfifth/DIS-840_display_descriptions

DIS-840: Display Descriptions

LeoStoyanov committed:
DIS-873: Update setting description.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-873-fix-and-refactor-koha-curbside-pickup

lathomas64 committed:
BWS-PKG - DIS-360: minor fix of additional formatting and conditional inclusion of title if data was missing

Abhyastamita committed:
DIS-720: Update release notes

LiYanjun19 committed:
DIS-908 Add special filter method

LiYanjun19 committed:
DIS-908 Override applyFilter() for addedByName

LiYanjun19 committed:
DIS-908 Override applyFilter() for user, library

LiYanjun19 committed:
DIS-908 Override applyFilter() for donationValue, donationComplete, dateCompleted

LiYanjun19 committed:
DIS-908 Override applyFilter() for user, library, device

LiYanjun19 committed:
DIS-908 Override applyFilter() for owningUser

LiYanjun19 committed:
DIS-908 Add release Note

LiYanjun19 committed:
DIS-908 Remove compareValue false checks

LiYanjun19 committed:
DIS-909 Fix OverDrive triggering SIP authentication when log in via SSO

mdnoble73 committed:
Merge pull request #2541 from Abhyastamita/DIS-720_sorting_sierra_holds_by_date_placed

Dis 720 sorting sierra holds by date placed

mdnoble73 committed:
Merge pull request #2540 from LiYanjun19/DIS-909-OverDrive-SSO

DIS-909 Fix OverDrive triggering SIP authentication when log in via SSO

LeoStoyanov committed:
DIS-911: Only preform DB insertions and updates of translations if the user has "Translate Aspen" permission.

LeoStoyanov committed:
DIS-911: Add validation for 'lookfor' and 'searchIndex' parameters.

K-Alette committed:
DIS-801: Materials Request: Remember Assigned To and Status Filters

Add table for admin sticky filters
Update logic for applying/saving filters on the Manage Materials Requests admin page

Update release notes

Tested on my local instance of Aspen

NashvillePublicLibrary committed:
DIS-39 Update release notes

LeoStoyanov committed:
DIS-911: Release notes.

NashvillePublicLibrary committed:
Revert "DIS-39 Update release notes"

This reverts commit acf79180d192b60878ee8c8216e868026b17cb52.

NashvillePublicLibrary committed:
DIS-861 update release notes

LeoStoyanov committed:
DIS-911: Make `translate()` method more readable.

NashvillePublicLibrary committed:
Merge remote-tracking branch 'upstream/25.06.00' into 25.06.00-SnapPay

NashvillePublicLibrary committed:
DIS-39 Update release notes

NashvillePublicLibrary committed:
Update 25.06.00.MD

Commit:
DIS-914: Missed 'return' statement has been added to patronEligibleForILLRequests method

Commit:
DIS-914: Update release notes 25.06.00

mdnoble73 committed:
Merge pull request #2546 from thekesolutions/return_missed_value

DIS-914: Return missed value

mdnoble73 committed:
Merge pull request #2451 from LeoStoyanovByWater/DIS-759-validate-before-removing-themes-and-libraries-from-themes

DIS-759: Validate Before Removing Themes and Libraries from Themes

lathomas64 committed:
Merge branch '25.06.00' into DIS-360-Load-both-title-and-body-for-Evergreen-copy-notes

mdnoble73 committed:
DIS-759 Properly apply class for additional list actions

mdnoble73 committed:
Merge branch '25.06.00' into DIS-819-Sierra-Self-Reg-Welcome-Email

mdnoble73 committed:
Merge pull request #2482 from LiYanjun19/DIS-819-Sierra-Self-Reg-Welcome-Email

DIS-819 Allow Sierra Self-Reg to send welcome email to patrons

K-Alette committed:
Update ManageRequests.php

Handle unassigned value and passing in no values for filters

K-Alette committed:
Merge branch '25.06.00' of https://github.com/K-Alette/aspen-discovery into 25.06.00

K-Alette committed:
Merge branch 'Aspen-Discovery:25.06.00' into 25.06.00

K-Alette committed:
Update ManageRequests.php

K-Alette committed:
Merge branch '25.06.00' of https://github.com/K-Alette/aspen-discovery into 25.06.00

mdnoble73 committed:
Merge pull request #2462 from LeoStoyanovByWater/DIS-786-show-list-titles-and-descriptions-when-printing-lists

DIS-786: Show List Titles & (Optional) Descriptions When Printing Lists

mdnoble73 committed:
DIS-786 Merge Javascript

mdnoble73 committed:
Merge pull request #2548 from K-Alette/25.06.00

DIS-801: Materials Request: Remember Assigned To and Status Filters

mdnoble73 committed:
Merge branch '25.06.00' of https://github.com/Aspen-Discovery/aspen-discovery into 25.06.00

mdnoble73 committed:
DIS-801 Add checking to ensure that at least one status, assignee, and format is selected when filtering requests

mdnoble73 committed:
Merge pull request #2489 from LeoStoyanovByWater/DIS-832-slight-refactor-of-oaiindexermain

DIS-832: Slight Refactor of OaiIndexerMain

mdnoble73 committed:
Merge pull request #2495 from openfifth/DIS-814_koha_login_error_messages

DIS-814: koha login error messages

mdnoble73 committed:
DIS-832 rebuild OAI Indexer

mdnoble73 committed:
Merge branch '25.06.00' of https://github.com/Aspen-Discovery/aspen-discovery into 25.06.00

mdnoble73 committed:
DIS-918 Fix LibKey notice

mdnoble73 committed:
Merge pull request #2505 from LeoStoyanovByWater/DIS-859-update-and-fix-generated-citations

DIS-859: Update and Fix Generated Citations

mdnoble73 committed:
Merge branch '25.06.00' into DIS-865-clear-self-reg-form-information-upon-back-page

mdnoble73 committed:
Merge pull request #2506 from LeoStoyanovByWater/DIS-865-clear-self-reg-form-information-upon-back-page

DIS-865: Clear Self-Reg Form Fields When Page Is Restored from History

mdnoble73 committed:
Merge pull request #2507 from LeoStoyanovByWater/DIS-848-reposition-expand-admin-sidebar-button

DIS-848: Reposition & Style Expand Admin Sidebar Button

mdnoble73 committed:
Merge branch '25.06.00' into restrict-isbn-input-for-materials-requests

mdnoble73 committed:
Merge pull request #2511 from LeoStoyanovByWater/restrict-isbn-input-for-materials-requests

DIS-868: Prevent Non-Digit ISBN/ISSN/UPC Input & Null‐Mapping Errors

mdnoble73 committed:
Merge branch '25.06.00' into DIS-493-prioritize-available-titles-from-koha-25.06.00

mdnoble73 committed:
Merge pull request #2510 from LeoStoyanovByWater/DIS-493-prioritize-available-titles-from-koha-25.06.00

DIS-493: Added Option to Prioritize Titles from Available Koha Records for Grouped Works Display Titles (25.06.00)

mdnoble73 committed:
Merge pull request #2520 from lathomas64/DIS-360-Load-both-title-and-body-for-Evergreen-copy-notes

Dis 360 load both title and body for evergreen copy notes

mdnoble73 committed:
Merge branch '25.06.00' into DIS-891-fix-incorect-copies-available-count-on-grouped-works

mdnoble73 committed:
Merge pull request #2521 from LeoStoyanovByWater/DIS-891-fix-incorect-copies-available-count-on-grouped-works

DIS-891: Incorrect Copies Available Count Displayed on Grouped Works

mdnoble73 committed:
Merge branch 'DIS-836_admin_control_over_ce_leaderboard' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-836_admin_control_over_ce_leaderboard

mdnoble73 committed:
Merge branch 'openfifth-DIS-836_admin_control_over_ce_leaderboard' into 25.06.00

mdnoble73 committed:
Merge branch 'DIS-851_do_not_duplicate_campaigns' of https://github.com/openfifth/aspen-discovery into openfifth-DIS-851_do_not_duplicate_campaigns

mdnoble73 committed:
Merge branch 'openfifth-DIS-851_do_not_duplicate_campaigns' into 25.06.00

mdnoble73 committed:
Merge branch '25.06.00' into DIS-893-suppress-copies-available-count-if-only-on-order-copies-exist

mdnoble73 committed:
Merge pull request #2523 from LeoStoyanovByWater/DIS-893-suppress-copies-available-count-if-only-on-order-copies-exist

DIS-893: Suppress Copies Available Count If Only On Order Copies Exist

mdnoble73 committed:
Merge branch '25.06.00' into DIS-900-show-subcategories-when-searching-admin-interface

mdnoble73 committed:
Merge pull request #2529 from LeoStoyanovByWater/DIS-900-show-subcategories-when-searching-admin-interface

DIS-900: Fixed Admin Search to Include All Menu Items

mdnoble73 committed:
Merge javascript

mdnoble73 committed:
Merge branch 'prevent_login_expired_cards' of https://github.com/thekesolutions/aspen-discovery into thekesolutions-prevent_login_expired_cards

mdnoble73 committed:
Merge branch 'thekesolutions-prevent_login_expired_cards' into 25.06.00

LeoStoyanov committed:
DIS-903: Move sleep logic.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-903-welcome-email-not-sent-when-registering-new-user-symphony

mdnoble73 committed:
Merge pull request #2537 from LeoStoyanovByWater/DIS-901-fix-grouped-work-user-rating-filtering

DIS-901: Fixed Grouped Work "Rating" Facet Filtering/Sorting, Browse Category Searching, & Add to Browse Sorting

mdnoble73 committed:
Merge branch '25.06.00' into DIS-908-Special-Filter-Method

mdnoble73 committed:
Merge pull request #2539 from LiYanjun19/DIS-908-Special-Filter-Method

DIS-908 Add Special filter method for handling virtual properties

LeoStoyanov committed:
DIS-873: Fix typo.

mdnoble73 committed:
Merge pull request #2545 from LeoStoyanovByWater/DIS-911-prevent-db-insertions-of-translations-without-permission

DIS-911: Prevent Unauthorized Translation Term Insertion & Add Search Security

mdnoble73 committed:
Merge pull request #2544 from Nashville-Public-Library/25.06.00-DIS-861

25.06.00 DIS-861 Circulation reports

mdnoble73 committed:
Merge branch '25.06.00-SnapPay' of https://github.com/Nashville-Public-Library/aspen-discovery into Nashville-Public-Library-25.06.00-SnapPay

mdnoble73 committed:
Merge branch 'Nashville-Public-Library-25.06.00-SnapPay' into 25.06.00

mdnoble73 committed:
Merge branch '25.06.00' into DIS-903-welcome-email-not-sent-when-registering-new-user-symphony

mdnoble73 committed:
Merge pull request #2531 from LeoStoyanovByWater/DIS-903-welcome-email-not-sent-when-registering-new-user-symphony

DIS-903: Fixed Welcome Email Not Being Sent When Registering New User (Symphony)

mdnoble73 committed:
Merge pull request #2517 from LeoStoyanovByWater/DIS-791-add-threads-to-update-reading-history-java-process

DIS-791: Enhance `UpdateReadingHistory` Cron with Configurable Thread Pool & Improved Logging

mdnoble73 committed:
Merge pull request #2468 from LeoStoyanovByWater/DIS-775-lazy-load-browse-categories-for-accessibility-layout

DIS-775: Improve Browse Category Page Load Speed for Accessible Mode & Fix Duplicate Items on Rapid Clicks

mdnoble73 committed:
DIS-775 Merge javascript

mdnoble73 committed:
Merge branch '25.06.00' into DIS-766-rework-reading-history

mdnoble73 committed:
Merge pull request #2459 from LeoStoyanovByWater/DIS-766-rework-reading-history

DIS-766: Rework Initial Reading History Loading Process

mdnoble73 committed:
DIS-923 Do not attempt to load holds and checkouts from Palace Project where Palace Project is not configured for them

mdnoble73 committed:
DIS-924 Correct notice when showing Other Editions and Formats.

mdnoble73 committed:
DIS-925 Remove unused code

mdnoble73 committed:
DIS-926 Performance improvements

mdnoble73 committed:
Add default readme and database update files

LeoStoyanov committed:
DIS-911: Display the default term if it does not exist yet in the translation_terms table.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-911-prevent-db-insertions-of-translations-without-permission

mdnoble73 committed:
Merge pull request #2550 from LeoStoyanovByWater/DIS-911-prevent-db-insertions-of-translations-without-permission

DIS-911: Display the Input Phrase If It Is Not a Translation Term Yet

mdnoble73 committed:
Merge release notes for 25.06

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

AlexanderBlanchardAC committed:
Merge branch '25.06.00' into DIS-838_padding_around_reward_images

mdnoble73 committed:
Merge pull request #2536 from openfifth/DIS-838_padding_around_reward_images

DIS-838: Padding Around Reward Images

mdnoble73 committed:
DIS-838 Correct release notes

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

LeoStoyanov committed:
DIS-873: Revert disabled-libraries change and instead display text if a library is already assigned to another Curbside Pickup setting.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-873-fix-and-refactor-koha-curbside-pickup

LeoStoyanov committed:
DIS-873: Fix spacing vs. tabs.

LeoStoyanov committed:
DIS-873: Add back DIS-818 release notes.

mdnoble73 committed:
Merge pull request #2538 from LeoStoyanovByWater/DIS-873-fix-and-refactor-koha-curbside-pickup

DIS-873: Fixed & Refactored Koha Curbside Pickup

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

LeoStoyanov committed:
DIS-911: Consider replacement values for dynamically injected phrases.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-911-prevent-db-insertions-of-translations-without-permission

mdnoble73 committed:
Merge pull request #2552 from LeoStoyanovByWater/DIS-911-prevent-db-insertions-of-translations-without-permission

DIS-911: Consider Replacement Values for Dynamically Injected Phrases

mdnoble73 committed:
Rebuild JavaScript and Jars in prep for UAT

LeoStoyanov committed:
DIS-493: Update DB maintenance to match naming elsewhere.

LeoStoyanov committed:
DIS-493: Update Java doc.

LeoStoyanov committed:
DIS-493: Drop old, incorrect column.

mdnoble73 committed:
Merge pull request #2553 from LeoStoyanovByWater/DIS-493-prioritize-available-titles-from-koha-25.06.00

DIS-493: Update DB Maintenance to Match Naming Elsewhere

mdnoble73 committed:
DIS-911 do not allow caching of translations with replacement values

mdnoble73 committed:
DIS-934 When a title exists in multiple series, ensure the volume from the correct series is shown when viewing a series

mdnoble73 committed:
Update active testers for 25.06

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

LeoStoyanov committed:
DIS-803: Narrow scope of selector, fix selector targeting, and fix layout clash with jQuery's validation errors.

LeoStoyanov committed:
Merge branch '25.06.00' into DIS-803-character-limit-feedback-when-editing-inputs-and-textareas

mdnoble73 committed:
DIS-847 - Update to allow saving read only fields for objects being inserted.

mdnoble73 committed:
DIS-911 Allow null phrases to be translated

mdnoble73 committed:
Merge pull request #2557 from LeoStoyanovByWater/DIS-803-character-limit-feedback-when-editing-inputs-and-textareas

DIS-803: Narrow Scope of Selector, Fix Selector Targeting, & Fix Layout Clashes with jQuery's Validation Errors

mdnoble73 committed:
DIS-803 merge javascript

mdnoble73 committed:
DIS-560 Expand column lengths to accommodate automatic naming within sideloads

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

mdnoble73 committed:
DIS-858 Correct quotes for oneToManyCell

Also corrects early

mdnoble73 committed:
Merge branch '25.06.00' into 25.07.00

mdnoble73 committed:
DIS-321 correct tooltip quotes in library settings
